ocemr/templates/patient_queue.html

```diff
@@ -15,6 +15,13 @@
 {% endblock %}
 
 {% block content %}
+<DIV ID=extraHeader>
+Filter by visit type:
+{% ifnotequal filter_visit_type None %}<A HREF=/patient_queue/>All</A>{%else%}ALL{%endifnotequal%} /
+{% ifnotequal filter_visit_type "OUT" %}<A HREF=/patient_queue/?filter_visit_type=OUT>Outpatient</A>{%else%}Outpatient{%endifnotequal%} /
```

Comment:
I was originally thinking we could remap the urlpatterns to support the top level views as described in #44 
lab queue and med queue are pretty well separated as `/lab/`  and `/med/`. Does it make sense to do `/inpatient/`, `/outpatient`, and `/maternity`?
I think it might be challenging to maintain the view through url prefixes just as much as filter parameters.

What's going to be the cleanest?

Reply:
I suppose it depends on how people work. If there is a clean deliniation between people that work on outpatient and inpatients, perhaps adding Outpatient, Inpatient, and Maternity to the main menu makes the most sense.

For any person, administration or management-wise, that may mean more bouncing around to get a complete view of the facility without an "All" option too..

I toyed around with putting the link up in the main row above, but sort of settled on the current placement for now. I guess from an administrative point of view, I like defaulting to "all." but would want the sire to remember my patient filter as i click around and return to the Patient Queue. (Currently, it's forgtten when you navigate away then go back to the Patient Queue.)

Reply:
I'm fine with your suggestion too.

I didn't know where to put Patient **search** and **add** links when I removed the generic "Patient Queue" and added "Inpatient", "Outpatient", and "Maternity" up top.